src/pages/OrganisationPage.vue

```diff
@@ -92,3 +125,19 @@
     </div>
   </ContentContainer>
 </template>
+<style src="vue-multiselect/dist/vue-multiselect.css"></style>
+<style>
+  .multiselect__option--highlight,
+  .multiselect__option--highlight:after {
+    background: #155366;
+  }
+  .multiselect__option--selected.multiselect__option--highlight {
+    background: #81c3d6;
+  }
+  .multiselect__option--selected.multiselect__option--highlight:after {
+    background: #81c3d6;
+  }
+  .multiselect__option--group-selected.multiselect__option--highlight {
+    background: #81c3d6;
+  }
+</style>
```

Comment:
What if I need this used else where (very likely), do I have to copy paste all this for every usage?
Create a component that embodies this customisation so we can use it throughout https://vuejs.org/guide/components/attrs.html#nested-component-inheritance